packages/cmf/src/sagas/http.js

```diff
@@ -170,14 +170,15 @@ export function httpFetch(url, config, method, payload) {
  */
 export function* wrapFetch(url, config, method = HTTP_METHODS.GET, payload, options) {
 	const answer = yield call(httpFetch, url, config, method, payload);
-
-	if (!get(options, 'silent') && answer instanceof Error) {
+	const silent = get(options, 'silent');
+	if (silent !== true && answer instanceof Error) {
 		yield put({
 			error: { message: answer.data.message, stack: { status: answer.response.status } },
 			url,
 			config,
 			method,
 			payload,
+			options,
```

Comment:
Just a note, here people could send anything, like `IwantANotification` and `redirectMe`. If we pass options, should we not validate it then? For example
```
options: {
  notify,
  redirect,
}
```

But on the other hand if the option doesn't actually do anything inside CMF, maybe we shouldn't even document it?

Reply:
those 2 options will be fully integrated within cmf once notification and redirect will be cmf-modules

Reply:
Agree with @annilsson, if the options are not used by cmf code, it should not be specified in its doc.
If you do a cmf external module that consume that, this should be in this module documentation